<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contourView</class>
 <widget class="QWidget" name="contourView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QPushButton{ border-style: outset;
                        border-width: 4px;
                        border-radius: 7px;
                        border-color: darkgray;
                        font: normal 12px;
                           padding: 2px; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Contour Lines every </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select spacing in waves of conatour lines.  Set t0 0 for no contour lines.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>waves</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fillContourCB">
       <property name="text">
        <string>FillContour</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Show All Wavefronts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="histogram">
       <property name="text">
        <string>Histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showRuler">
       <property name="text">
        <string>Ruler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="LinkProfileCB">
       <property name="text">
        <string>Link to Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ContourPlot" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContourPlot</class>
   <extends>QWidget</extends>
   <header>contourplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
